Containers/AboutScreen.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {
+  Image,
+  Text,
+  View,
+} from 'react-native'
+import { connect } from 'react-redux'
+import { Images } from '../Themes'
+import styles from './Styles/AboutScreenStyle'
+
+class AboutScreen extends React.Component {
+  static navigationOptions = {
+    tabBarLabel: 'General Info',
+    tabBarIcon: ({ focused }) => (
+      <Image source={focused ? Images.activeInfoIcon : Images.inactiveInfoIcon} />
+    )
+  }
+
+  render () {
+    return (
+        <View style={styles.container}>
+          <Text style={styles.sectionText}>About Screen</Text>
+        </View>
+      )
+  }
+}
+
+const mapStateToProps = (state) => {
+  return {
+  }
+}
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+  }
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(AboutScreen)
```

Comment:
@jpknoll no reason to use redux at all on this screen, and in fact no reason to make it a component at all.  Pure function would do the job

Reply:
@srkirkland The original version was mapping a datetime redux to this screen, but yeah, it should be a simple component at least.